xray/formats/simplejsonapi.go

```diff
@@ -30,7 +30,7 @@ type VulnerabilityOrViolationRow struct {
 	References               []string                  `json:"references"`
 	ImpactPaths              [][]ComponentRow          `json:"impactPaths"`
 	JfrogResearchInformation *JfrogResearchInformation `json:"jfrogResearchInformation"`
-	Technology               coreutils.Technology      `json:"-"`
+	Technology               coreutils.Technology
```

Comment:
Shouldn't we add it with the `json:"technology"` annotation, so that it is saved in the JSON starting with a lowe-case?